### ISSUE TITLE
[NPC] Elwynn Forest - Goldshire - Erma <Stable Master>: wrong interaction

### ISSUE THREAD

**Issue by ailas** [dalaranwow/dalaran-wow#5267]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: If you visit Erma <Stable Master> in Elwynn Forest - Goldshire and click on her like you want to talk to her, dialog shows option "I'm looking for a lost companion.". Click on that option and it disappears.

**Expected behaviour**: Continued interaction?

**Steps to reproduce the problem**:

1. Visit Goldshire
2. Talk to Erma <Stable Master> (she is standing next to Inn entrance)
3. Click on "I'm looking for a lost companion."

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

http://www.wowhead.com/npc=6749/erma

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello after a quick look on PTR the:

> "I'm looking for a lost companion."

is still present.
![image](https://user-images.githubusercontent.com/6797611/35529462-c953a768-0539-11e8-9155-ac6f1f4d692a.png)
after clicking on it:
![image](https://user-images.githubusercontent.com/6797611/35529473-d55ec16e-0539-11e8-9441-7ae84e356e35.png)
